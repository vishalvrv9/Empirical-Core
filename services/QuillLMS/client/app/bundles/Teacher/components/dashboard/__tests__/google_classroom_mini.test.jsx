import React from 'react';
import { shallow } from 'enzyme';

import GoogleClassroomMini from '../google_classroom_mini';

describe('GoogleClassroomMini component', () => {

  it('should render', () => {
    expect(shallow(<GoogleClassroomMini />)).toMatchSnapshot();
  });

});
