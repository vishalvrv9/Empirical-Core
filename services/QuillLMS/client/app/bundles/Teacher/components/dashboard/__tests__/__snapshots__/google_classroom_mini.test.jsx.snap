// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoogleClassroomMini component should render 1`] = `
<div
  className="mini_container col-md-4 col-sm-5 text-center"
>
  <div
    className="mini_content"
  >
    <div
      className="resources-container google-classroom-announcement"
    >
      <h4>
        Google Classroom Announcement
      </h4>
      <img
        src="/images/google_classroom_icon.png"
      />
      <p>
        Your students can now use Clever or Google Classroom to automatically join your classroom.
      </p>
      <span>
        <a
          href="https://medium.com/@quill/we-re-now-supporting-google-classroom-and-clever-55f2c12272f7#.xfrtabflg"
        >
          Read the Announcement &gt;
        </a>
      </span>
    </div>
  </div>
</div>
`;
